docs/source-app/installation_mac.rst

```diff
@@ -0,0 +1,22 @@
+:orphan:
+
+#############################
+Installation on Apple M1 Macs
+#############################
+
+Apple M1 Mac environments need a bit of tweaking before you install.
+
```

Comment:
@awaelchli will the same apply for M3?

I'm thinking this will be temporary and we will do something "soon" so this can actually be removed from the docs.

Reply:
@Felonious-Spellfire Yes, it should be the same for M1, M2, and if M3  is released:)

Reply:
@awaelchli @manskx 

Updated